cli.js

```diff
@@ -133,6 +133,10 @@ st.on('done', function () {
 });
 
 st.on('error', function (err) {
-	console.error(err);
+	if (err.code === 'ENOTFOUND') {
+		console.log(chalk.bold.red('Error:'), 'please check your internet connection.');
```

Comment:
How much improvements can you have at so minimal code change :).

Thanks! All valid points off course.